frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.test.tsx

```diff
@@ -87,6 +97,13 @@ describe('PageConfigPanel', () => {
     const uniqueIds = screen.queryByText(/<idcontainer>, <idContainer3>/i);
     expect(uniqueIds).not.toBeInTheDocument();
   });
+
+  it('should show warning modal when there are duplicated ids in all layouts', async () => {
+    renderPageConfigPanel();
+
+    const modal = screen.getByRole('dialog', { hidden: true });
```

Comment:
Is it possible to make this test without `{ hidden: true }`?

Reply:
I think we need` { hidden: true `} here because our modal is not initially visible in the UI; it relies on the `hasDuplicatedIdsInAllLayouts `condition to determine if it should be shown. The modal is hidden by default. Using` { hidden: true }` in the test allows us to check if the modal is present in the DOM, even if it's not currently visible.

Reply:
fixed